human_player/lib/human_player/impl.ex

```diff
@@ -0,0 +1,131 @@
+defmodule HumanPlayer.Impl do
```

Comment:
I like the small functions. Perhap this module could have been broken up into a number of smaller modules?